.drone.yml

```
@@ -0,0 +1,116 @@
+pipeline:
+
+  build-docker-image:
+    image: docker:17.09.1
+    environment:
+      - DOCKER_HOST=tcp://172.17.0.1:2375
+    commands:
+      - docker build -t hocs-management-ui .
+    when:
+      branch: [master, feature/HOCS-1022-create-build-pipeline]
+      event: push
+
+  install-docker-image:
+    image: docker:17.09.1
+    environment:
+      - DOCKER_HOST=tcp://172.17.0.1:2375
+    secrets:
+      - docker_password
+    commands:
+      - docker login -u="ukhomeofficedigital+hocs" -p=$${DOCKER_PASSWORD} quay.io
+      - docker tag hocs-management-ui quay.io/ukhomeofficedigital/hocs-management-ui:build-$${DRONE_BUILD_NUMBER}
+      - docker tag hocs-management-ui quay.io/ukhomeofficedigital/hocs-management-ui:latest
+      - docker push quay.io/ukhomeofficedigital/hocs-management-ui:build-$${DRONE_BUILD_NUMBER}
+      - docker push quay.io/ukhomeofficedigital/hocs-management-ui:latest
+    when:
+      branch: [master, feature/HOCS-1022-create-build-pipeline]
```

```suggestion
      branch: master
```
You probably either want this to push to Quay on every build, or only when things get merged into master (so that every build on Quay theoretically works)

---

Removed specific feature branch and added in dev branch which we'll be using from now on.